cal/cal/models.py

```diff
@@ -583,7 +583,7 @@ def query(self, calendar_ids=None, start=None, end=None):
         if end:
             events_qs = events_qs.filter(start__lt=end)
         else:
-            events_qs = events_qs.filter(start__lt=datetime.utcnow())
+            events_qs = events_qs.filter(start__lt=datetime.now(pytz.utc))
```

Comment:
yay Python. naw idk I figured it out cuz the graph and events didn't show up, then I clicked something and it showed an error page.